### PR TITLE
update HCC version string to include commit from llvm monorepo

### PR DESCRIPTION
**Pull request by david-salinas**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -208,9 +208,9 @@ if (NOT HCC_VERSION_STRING)
 endif()
 
 # Set HCC version string. The rule for version string is:
-# HCC_VERSION_MAJOR . HCC_VERSION_MINOR . HCC_VERSION_PATCH-KALMAR_SDK_COMIT-KALMAR_FRONTEND_COMMIT-KALMAR_BACKEND_COMMIT
+# HCC_VERSION_MAJOR . HCC_VERSION_MINOR . HCC_VERSION_PATCH-KALMAR_SDK_COMIT-LLVM_COMMIT
 add_version_info_from_git(HCC_VERSION_STRING
-  HCC_VERSION_PATCH KALMAR_SDK_COMMIT KALMAR_FRONTEND_COMMIT KALMAR_BACKEND_COMMIT)
+  HCC_VERSION_PATCH KALMAR_SDK_COMMIT KALMAR_LLVM_COMMIT)
```
Comment (scchan) [inline CMakeLists.txt]: could you change to HCC_LLVM_COMMIT?

File: CMakeLists.txt [hunk 2]
```diff
@@ -208,9 +208,9 @@ if (NOT HCC_VERSION_STRING)
 endif()
 
 # Set HCC version string. The rule for version string is:
-# HCC_VERSION_MAJOR . HCC_VERSION_MINOR . HCC_VERSION_PATCH-KALMAR_SDK_COMIT-KALMAR_FRONTEND_COMMIT-KALMAR_BACKEND_COMMIT
+# HCC_VERSION_MAJOR . HCC_VERSION_MINOR . HCC_VERSION_PATCH-KALMAR_SDK_COMIT-LLVM_COMMIT
```
Comment (scchan) [inline CMakeLists.txt]: This doesn't match the variable name used below

**Review by scchan — CHANGES_REQUESTED** [review]
do you have a corresponding change for llvm-project?

**Comment by david-salinas** [comment]
> do you have a corresponding change for llvm-project?

I do.  In gerrit.  I'll add you as a reviewer

File: CMakeLists.txt (hunk 1 above)
Comment (david-salinas) [reply CMakeLists.txt]: yes.  done in the latest update.

File: CMakeLists.txt (hunk 2 above)
Comment (david-salinas) [reply CMakeLists.txt]: Do you mean, at line #288? where we set "KALMAR_BACKEND" ?  I don't think that variable is used in the version string.

**Review by scchan — APPROVED** [review]

**Pull request merged**